content/influxdb/v2.1/reference/syntax/line-protocol.md

```diff
@@ -291,31 +305,27 @@ The [escaping rules](#escaping-rules) imply the following for [tag keys](/influx
 - they accept double quote (`"`) and single quote (`'`) characters as part of the name, key, or value.
 
 In _string_ [field values](/influxdb/v2.1/reference/glossary/#field-value) with two contiguous backslashes, the first backslash is interpreted as an escape character.
-For example:
-
-| Backslashes | Interpreted as |
-|:-----------:|:-------------:|
-| `\`         | `\`           |
-| `\\`        | `\`           |
-| `\\\`       | `\\`          |
-| `\\\\`      | `\\`          |
-| `\\\\\`     | `\\\`         |
-| `\\\\\\`    | `\\\`         |
 
 Given the following line protocol:
 
 ```py
+# Escaped = in tag value.
+# Literal backslash at start of string field value.
+# Escaped backslash at end of string field value.
+airSensor,sensor_id=TLM\=0201 desc="\=My data==\\"
+
 # Measurement name with literal backslashes.
-# Escaped = in tag value and \ in string field value.
-air\\\\\Sensor,sensor_id=TLM\=0201 temp=70.0,humidity=0.5,desc="\\\\==My data\==\\\"
+# Escaped = in tag value.
+# Escaped \ and escaped " in string field value.
+air\\\\\Sensor,sensor_id=TLM\=0201 desc="\\\"==My data\==\\"
```

Comment:
to be sure, three backslashes in a row, results in one written as the value?

Reply:
Yes, depending on the character following the last slash (which is why I removed that table). In this case, you get one escaped `\` and an escaped `"`.